comid/example_test.go

```diff
@@ -161,7 +161,7 @@ func Example_encode() {
+	// a50065656e2d474201a10078206d792d6e733a61636d652d726f616472756e6e65722d737570706c656d656e740282a3006941434d45204c74642e01d8207468747470733a2f2f61636d652e6578616d706c6502820100a20069454d4341204c74642e0281020382a200781a6d792d6e733a61636d652d726f616472756e6e65722d626173650100a20078196d792d6e733a61636d652d726f616472756e6e65722d6f6c64010104a5008182a300a500d86f445502c000016941434d45204c74642e026a526f616452756e6e65720300040101d902264702deadbeefdead02d8255031fb5abf023e4992aa4e95f9c1503bfa81a200d8255031fb5abf023e4992aa4e95f9c1503bfa01aa01d90228020282820644abcdef00820644ffffffff03a201f403f504d9023044010203040544ffffffff064802005e1000000001075020010db8000000000000000000000068086c43303258373056484a484435094702deadbeefdead0a5031fb5abf023e4992aa4e95f9c1503bfa018182a300a500d8255031fb5abf023e4992aa4e95f9c1503bfa016941434d45204c74642e026a526f616452756e6e65720300040101d902264702deadbeefdead02d8255031fb5abf023e4992aa4e95f9c1503bfa81a200d8255031fb5abf023e4992aa4e95f9c1503bfa01aa01d90229020282820644abcdef00820644ffffffff03a300f401f403f504d9023044010203040544ffffffff064802005e1000000001075020010db8000000000000000000000068086c43303258373056484a484435094702deadbeefdead0a5031fb5abf023e4992aa4e95f9c1503bfa028182a101d902264702deadbeefdead81d9022a78b12d2d2d2d2d424547494e205055424c4943204b45592d2d2d2d2d0a4d466b77457759484b6f5a497a6a3043415159494b6f5a497a6a304441516344516741455731427671462b2f727938425761375a454d553178595948455138420a6c4c54344d46484f614f2b4943547449767245654570722f7366544150363648326843486462354845584b74524b6f6436514c634f4c504131513d3d0a2d2d2d2d2d454e44205055424c4943204b45592d2d2d2d2d038182a101d8255031fb5abf023e4992aa4e95f9c1503bfa81d9022a78b12d2d2d2d2d424547494e205055424c4943204b45592d2d2d2d2d0a4d466b77457759484b6f5a497a6a3043415159494b6f5a497a6a304441516344516741455731427671462b2f727938425761375a454d553178595948455138420a6c4c54344d46484f614f2b4943547449767245654570722f7366544150363648326843486462354845584b74524b6f6436514c634f4c504131513d3d0a2d2d2d2d2d454e44205055424c4943204b45592d2d2d2d2d08818282a300a500d86f445502c000016941434d45204c74642e026a526f616452756e6e65720300040101d902264702deadbeefdead02d8255031fb5abf023e4992aa4e95f9c1503bfa81a200d8255031fb5abf023e4992aa4e95f9c1503bfa01aa01d90228020282820644abcdef00820644ffffffff03a201f403f504d9023044010203040544ffffffff064802005e1000000001075020010db8000000000000000000000068086c43303258373056484a484435094702deadbeefdead0a5031fb5abf023e4992aa4e95f9c1503bfaa16656616c756573818281a200d8255031fb5abf023e4992aa4e95f9c1503bfa01a501d90228020282820644abcdef00820644ffffffff03a201f403f504d9023044010203040544ffffffff81a200d8255031fb5abf023e4992aa4e95f9c1503bfa01a3064802005e1000000001075020010db8000000000000000000000068094702deadbeefdead
```

Comment:
this is not correct: it still produces the spurious `”Values"` key:
```
{
  "Values":[
    [
      [
        {
          0:37(h'31fb5abf023e4992aa4e95f9c1503bfa'),
          1:{
            1:552(2),
            2:[[6, h'abcdef00'], [6, h'ffffffff']],
            3:{1: false, 3: true},
            4:560(h'01020304'),
            5:h'ffffffff'
          }
        }
      ],
      [
        {
          0:37(h'31fb5abf023e4992aa4e95f9c1503bfa'),
          1:{
            6:h'02005e1000000001',
            7:h'20010db8000000000000000000000068',
            9:h'02deadbeefdead'
          }
        }
      ]
    ]
  ]
}
```

Reply:
> this is not correct: it still produces the spurious `”Values"` key:
> 
> ```
> {
>   "Values":[
>     [
>       [
>         {
>           0:37(h'31fb5abf023e4992aa4e95f9c1503bfa'),
>           1:{
>             1:552(2),
>             2:[[6, h'abcdef00'], [6, h'ffffffff']],
>             3:{1: false, 3: true},
>             4:560(h'01020304'),
>             5:h'ffffffff'
>           }
>         }
>       ],
>       [
>         {
>           0:37(h'31fb5abf023e4992aa4e95f9c1503bfa'),
>           1:{
>             6:h'02005e1000000001',
>             7:h'20010db8000000000000000000000068',
>             9:h'02deadbeefdead'
>           }
>         }
>       ]
>     ]
>   ]
> }
> ```

yes identified the bug, already and working on it!